CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
+# 76.1.1
+
+Fix build failures.
+
 # 76.1.0
 
-Add additional snapshot tests and mark outdated theming API as deprecated.
+Add additional snapshot tests and mark previous Dialog Themer APIs as deprecated.
```

Comment:
Changing these won't have any effect on the tagged 76.1.0 release and would make the CHANGELOG no longer be an "append" operation for each release. I suggest we leave the old version as-is and focus this release on any fixes to Cocoapods.